src/staff/Staff.tsx

```diff
@@ -2,21 +2,20 @@ import React from "react";
 
 import "./Staff.css";
 
-export type StaffProps = {
-  x?: number;
-  y?: number;
-  viewBox?: string;
-  height?: string;
-  width?: string;
-};
+/**
+ * We're extending Staff with SVGAttributes to keep DRY
+ */
```

Comment:
There is no need to add comments explaining the code, maybe with a few exceptions. Would you mind removing this comment?

Reply:
Sure!! Will remove the comments aS WELL!

Reply:
BTW: I'm implementing interfaces from now on, so I think is a good thread to start :)

Reply:
Removed this changes in e204311972269c7afcc78c494796eaae78fc659c, Amy!